cellprofiler/gui/help.py

```diff
@@ -2118,8 +2118,13 @@
         process the images in all movies together as if they were constituents of only one movie.
     </li>
 </ul>
+If your images represent a volume, you can follow the above example to process your volume. It is important to note,
+however, that CellProfiler will analyze each image individually. If you would like to analyze your data as a whole
+volume, we suggest building a .TIF stack from the individual image files using another software application, like FIJI.
+Additionally, whole volume processing is only supported for single-channel volumes. If your images are multi-channel
+you will need to separate the channels before combining images into a single-image stack.
```

Comment:
@bethac07 @karhohs @AnneCarpenter Can you please verify the above paragraph makes sense?